vignettes/methodological_introduction.Rmd

```diff
@@ -0,0 +1,142 @@
+---
+title: "What are MMRMs? A brief introduction"
+package: mmrm
+#bibliography: '`r system.file("REFERENCES.bib", package = "mmrm")`'
+#bibliography: references.bib # temporary bibliography
+csl: jrss_style.csl
+output:
+  rmarkdown::html_document:
+          theme: "spacelab"
+          highlight: "kate"
+          toc: true
+          toc_float: true
+vignette: |
+  %\VignetteIndexEntry{What are MMRMs? A brief introduction}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown} 
+editor_options:
+  chunk_output_type: console
+  markdown:
+    wrap: 72
+---
+
+## Abstract
+
+Mixed models for repeated measures (MMRMs) are frequently used in the analysis of data from clinical trials. 
+They are specifically suited to model continuous variables that were repeatedly measured at discrete time points (or within defined time-windows).
+In a clinical trial, these time points are typically visits according to a schedule that is pre-defined in the trial protocol.
+The distinguishing feature of MMRMs, compared to other implementations of linear mixed models, is that subject-specific random effects (which are not of direct interest for estimation and inference) are considered as residual effects, i.e. they are part of the error correlation matrix.
+This vignette provides a brief methodological introduction to MMRMs.<!--, largely irrespective of issues related to their technical implementation-->. 
```

Comment:
please remove comments in next step. If follow on work might be needed, please file separate issue on github, i.e. no TODOs left here please

Reply:
Done